src/DiscoverMembers.js

```diff
@@ -0,0 +1,54 @@
+import Account from './Card';
+
+
+function DiscoverMembers(){
+    return (
+        <div>
+            <div className='flex flex-row justify-around flex-wrap w-100'>
+              <div>
+                <Account label1="A Perera"/>
+              </div>
+              <div>
+                <Account label1="B Perera"/>
+              </div>
+              <div>
+                <Account label1="C Perera"/>
+              </div>
+
+              <div>
+                <Account label1="D Perera"/>
+              </div>
+              <div>
+                <Account label1="E Perera"/>
+              </div>
+              <div>
+                <Account label1="F Perera"/>
+              </div>
+              <div>
+                <Account label1="G Perera"/>
+              </div>
+              <div>
+                <Account label1="H Perera"/>
+              </div>
+              <div>
+                <Account label1="I Perera"/>
+              </div>
+              <div>
+                <Account label1="G Perera"/>
+              </div>
+              <div>
+                <Account label1="H Perera"/>
+              </div>
+              <div>
+                <Account label1="I Perera"/>
+              </div>
```

Comment:
Add a list of these names and map it with account card

Reply:
Done